Tests/XcodeGenKitTests/SchemeGeneratorTests.swift

```diff
@@ -182,32 +182,35 @@ class SchemeGeneratorTests: XCTestCase {
             }
 
             $0.it("generates target schemes from config variant") {
-                let configVariants = ["Test", "Production"]
+                let configVariants = ["Test", "PREPROD", "PROD"]
                 var target = app
                 target.scheme = TargetScheme(configVariants: configVariants)
                 let configs: [Config] = [
                     Config(name: "Test Debug", type: .debug),
-                    Config(name: "Production Debug", type: .debug),
+                    Config(name: "PREPROD Debug", type: .debug),
+                    Config(name: "PROD Debug", type: .debug),
                     Config(name: "Test Release", type: .release),
-                    Config(name: "Production Release", type: .release),
+                    Config(name: "PREPROD Release", type: .release),
```

Comment:
I mean if you were to change this to `Prepod release` I think you'd get a crash as the config can't be found with the variant name